src/layer.ts

```diff
@@ -130,6 +141,22 @@ export function applyExtensionLayer(service: Service, handlers: FunctionInfo[],
   }
 }
 
+export function applyDotnetTracingLayer(service: Service, handler: FunctionInfo, layers: LayerJSON) {
+  const { region } = service.provider;
+  const regionRuntimes = layers.regions[region];
+  if (regionRuntimes === undefined) {
+    return;
+  }
+
+  const traceLayerKey: string = "dotnet";
```

Comment:
Nit: I would define this constant outside of the function, since it is always the same. So if you call the function five times, there's no reason to define the same constant five times. Note that if you move it outside of the function, you might want to rename it slightly to make it more clear what it's for.